core/src/main/java/jenkins/ClassLoaderReflectionToolkit.java

```diff
@@ -4,41 +4,29 @@
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.net.URL;
+import java.net.URLClassLoader;
 import java.util.Enumeration;
 import edu.umd.cs.findbugs.annotations.CheckForNull;
 import edu.umd.cs.findbugs.annotations.NonNull;
+import jenkins.util.AntClassLoader;
+import jenkins.util.AntWithFindResourceClassLoader;
 
 /**
  * Reflective access to various {@link ClassLoader} methods which are otherwise {@code protected}.
+ *
+ * Initially tries to access methods using known classloaders in use that expose the methods
+ * to prevent IllegalReflectiveAccess errors on Java 11+
```

Comment:
Is this supposed to be a class name? If so, use `@link` or (if 9+) `@code`. Or some other code name? `@code`. Otherwise, avoid CamelCase.

Reply:
```suggestion
 * to prevent illegal reflective access errors on Java 11+
```